cns/service.go

```diff
@@ -190,6 +191,18 @@ func getTLSConfigFromFile(tlsSettings localtls.TlsSettings) (*tls.Config, error)
 		},
 	}
 
+	if tlsSettings.UseMTLS {
+		rootCAs, err := mtlsRootCAsFromCertificate(&tlsCert)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to get root CAs for configuring mTLS")
+		}
+		tlsConfig.ClientAuth = tls.RequireAndVerifyClientCert
+		tlsConfig.ClientCAs = rootCAs
+		tlsConfig.RootCAs = rootCAs
+	}
+
+	log.Debugf("TLS configured successfully from file: %+v", tlsSettings)
```

Comment:
just curious, do we have structure logging for this? what is the difference between logger.Debugf and log.Debugf there,

Reply:
CNS still mostly does not use structured logging. Here, `log` is the ACN logger, `logger` is the CNS logger...we should probably prefer `logger`?

Reply:
@rbtr  do we have plan to move logger to zap-logger for CNS?

Reply:
there's no concrete plan for the work to happen, right now. CNS actually [already has a zap logger](https://github.com/Azure/azure-container-networking/blob/e491f20ae07df49c2962aee1d75a3170ec9dfc23/cns/service/main.go#L631-L636) that is used in a few places, but there is a much bigger migration that needs to happen where we port the existing `logger` to use a zap/zapai backend.

Reply:
let me change this to use `logger`, thanks for the context around that here. do we have a way to also emit metrics? this would be useful for cases where we want to see a rollout of TLS/mTLS for specific partners/scenarios.

Reply:
refactored to use `logger`